ckan/templates/user/read_organizations.html

```diff
@@ -1,7 +1,7 @@
 {% extends "user/read_base.html" %}
 
 {% set user = user_dict %}
-{% set group_type = h.default_group_type('organization') %}
+{% set org_type = h.default_group_type('organization') %}
```

Comment:
From experimenting with my local instance just now, this line is not necessary in this template at all: if `org_type` and `group_type` are set here, they are in any case overwritten by the equivalent lines in the parent template `user/read_base.html`, and those values are used. This is what causes the bug that, in `primary_content_inner`, `group_type` is set to `"group"`, and therefore the messages are incorrect.

(This was not how I expected variable setting in an extending Jinja2 template to work, and presumably not how the original writer of this template thought it worked either.)

Given that, I would propose removing this `set` entirely as it is just confusing, and keeping the change of variable name within `primary_content_inner` so that `humanize_entity_type` is called with the right value `"organization"`.

I also noticed that I missed correcting that in the `<h2>`, so I will change that as well.